base/tags.ts

```diff
@@ -1,4 +1,5 @@
 export const LONG = '@long';
 export const SHORT = '@short';
 export const STAGE = '@stage';
-export const SMOKE = '@smoke';
\ No newline at end of file
+export const SMOKE = '@smoke';
+export const BROWSER = '@browserTest';
```

Comment:
```suggestion
export const BROWSER = '@browser';
```

Reply:
Verstehe den Namen browser noch nicht so ganz, sind die chroium tests headless?

Reply:
Die Tests werden immer headless, ausgeführt, dieses ist die default-Ausführung. Bislang benutzen wir nur chromium. Jetzt kommt firefox hinzu. Die Tests werden jetzt immer in 2 browsern ausgeführt, das wollen wir nicht da dann ja zb. short die doppelte Anzahl an tests hätte. Darum gibt es in yml den Parameter: --project "chromium".

In der neuen pipeline ...browserTest sieht das so aus: npx playwright test -g "@browserTest" --project "firefox"
Es werden in dieser pipeline nur ausgewählte Tests in firefox ausgeführt; in den anderen pipelines bleibt alles so wie es ist.

Reply:
Dann finde ich "browser" irreführend. Wenn Firefox der einzige Unterschied ist, dann lieber firefox nennen.

Reply:
browsertest steht für firefox, edge, safari... Das richtige Wort ist wohl crossBrowserTest

Reply:
Ich glaube, so funktioniert das und macht Sinn?

Reply:
Das "Test" kannst du dann weglassen, das haben die anderen Tags auch nicht. Ist das Ziel, dass mehrere Browser mit diesem Tag getestet werden sollen? Dann finde ich crossBrowser sehr gut

Reply:
Ja, die ausgewählten Tests sollen in mehreren Browsern laufen.